Added information about source side validation script (#1477)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeServer/collaboration/public-folders/migrate-to-exchange-online.md

The scripts and files you're downloading are:

>  ` ssv.ps1`:  Source Side Validation script scans the public folders at source and reports issues found along with action to fix the issues. You'll run this script on the Exchange server On-Premises.

Reviewer: ```suggestion
 ` ssv.ps1`:  Source Side Validation script scans the public folders at source and reports issues found along with actions required to fix the issues. You'll run this script on the Exchange server On-Premises.
```

## Step 2: Prepare for the migration

> Add a step here suggesting to run Source Side Validation script at mailbox server on Exchange Server 2010. Please use the examples as documented here: https://techcommunity.microsoft.com/t5/Exchange-Team-Blog/Making-your-public-folder-migrations-faster-and-more-reliable/ba-p/917622. The script will perform following pre-requisites:

Reviewer: ```suggestion
We recommend running Source Side Validation script from a mailbox server on Exchange Server 2010. Please use the examples as documented [here](https://techcommunity.microsoft.com/t5/Exchange-Team-Blog/Making-your-public-folder-migrations-faster-and-more-reliable/ba-p/917622).
```
